docs/debug-distributed-application.md

```diff
@@ -218,3 +251,33 @@ application is behaving. They provide a cheap, cost-effective way of getting a p
 
 > TODO: describe useful metrics/patterns and correlation, specifically related to Zeebe, that we use when debugging
 
+## JFR
+
+To run JFR inside a container we can make use of the following command:
+
```

Comment:
🙃 Ideally we have some more explanations about the tools, but I need to do this a bit later. This should be just the starting point.